server/meshmodel/components/core/WASMFilter.json

```diff
@@ -0,0 +1,33 @@
+{
+    "kind": "WASMFilter",
+    "apiVersion": "core.meshmodel.dev/v1alpha1",
+    "displayName": "WASM Filter",
+    "format": "JSON",
+    "isAnnotation": true,
+    "metadata": {
+        "logo-URL": "",
```

Comment:
Change to logoURL to maintain our camelcasing design principle. If we have done this in other components, hopefully we don’t have logic expecting a hyphen.

Reply:
@Abhishek-kumar09 do you know?